tests/tests/market_tests.cpp

```diff
@@ -1817,13 +1817,13 @@ BOOST_AUTO_TEST_CASE(mcr_bug_cross1270)
    auto mi = db.get_global_properties().parameters.maintenance_interval;
    generate_blocks(HARDFORK_CORE_1270_TIME - mi);
 
-   const asset_object& core = get_asset("BTS");
+   const asset_object& core = get_asset(GRAPHENE_SYMBOL);
    const asset_object& bitusd = get_asset("USDBIT");
    const asset_id_type bitusd_id = bitusd.id;
    const account_object& feedproducer = get_account("feedproducer");
 
    // feed is expired
-   BOOST_CHECK_EQUAL((*bitusd_id(db).bitasset_data_id)(db).current_feed.maintenance_collateral_ratio, 1750);
+   BOOST_CHECK_EQUAL((*bitusd_id(db).bitasset_data_id)(db).current_feed.maintenance_collateral_ratio, GRAPHENE_DEFAULT_MAINTENANCE_COLLATERAL_RATIO);
    BOOST_CHECK_EQUAL((*bitusd_id(db).bitasset_data_id)(db).feed_is_expired(db.head_block_time()), false); // should be true?
```

Comment:
The check `feed_is_expired(...)` doesn't make sense, please update or remove it.

When all feeds are expired, current_feed would be reset to default (which would be the new value and would not be expired).

Reply:
thank you for the detailed explication. I removed the checks here https://github.com/bitshares/bitshares-core/pull/1594/commits/db1699c5ee8c84bcf604483709aafbd76af26171